test/kind/init/tests/00-install.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.crt /tmp/ca.crt
+  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.key /tmp/ca.key
```

Comment:
this makes a strong assertion that this is a kind cluster and kind tests only..   this is under "kind" folder... these will not be able to used across clusters... it might be good to have comments here or a doc in the folder that sets assumptions and expectations for these tests.

Reply:
after some more thought... I don't understand these tests... this is under "kind" folder so it isn't surprising that they are kind specific and kind only... but I don't know why we would test kind.    Is this testing konvoy components on kind?  or?

Reply:
This isn't really a test... It's setting up stuff. I'm just cheating and using kuttl to do it hence `kind/init`.

I originally had the yaml files in `kind/init` but they weren't getting picked up, so I moved them into "tests" so they would get picked up and executed.